src/components/calcite-fab/calcite-fab.e2e.ts

```diff
@@ -0,0 +1,58 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { accessible, hidden, renders } from "../../tests/commonTests";
+
+describe("calcite-fab", () => {
+  it("renders", async () => renders("calcite-fab"));
+
+  it("honors hidden attribute", async () => hidden("calcite-fab"));
+
+  it("should have visible text when text is enabled", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-fab text="hello world" text-enabled></calcite-fab>`);
+
+    const button = await page.find("calcite-fab >>> .button");
```

Comment:
@asangma  can we just import the `CSS` const and use the `CSS.button` var instead of `.button` here? It applies multiple times in these tests. Once that is there I think this is good to merge.

Reply:
That's cool with me. I think that was there when I got to it.